src/IconButtonToggle/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { Container } from '..';
+
+/**
+ *
+ * @param {object} args arguments
+ * @param {boolean} args.activeValue the current value of the active/selected iconButton.
+ *                                   if not provided, none of the iconButtons will initially be active
+ * @param {Function} args.onChange callback to call when toggle value changes.
+ *                                 Receives value of the selected toggle button.
+ * @param {Array<IconButton>} args.children children components expected to be IconButton
+ * @returns {React.Component} A React component
+ */
+const IconButtonToggle = ({ activeValue, onChange, children }) => {
+  const iconButtons = useMemo(
+    () => React.Children.map(children, iconButton => (
+      React.cloneElement(iconButton, {
+        onClick: () => { onChange(iconButton.props.value); },
+        isActive: iconButton.props.value === activeValue,
+        'data-testid': `icon-btn-val-${iconButton.props.value}`,
+      })
+    )),
+    [children, activeValue],
+  );
+  return <Container className="icon-button-toggle-container">{iconButtons}</Container>;
+};
+
+IconButtonToggle.defaultProps = {
+  onChange: () => {},
+  activeValue: undefined,
+};
+
+IconButtonToggle.propTypes = {
+  activeValue: PropTypes.any, // eslint-disable-line react/forbid-prop-types
```

Comment:
nit: Let's add comment descriptions above each of the prop types so they show up in the documentation site's props table.